### PR TITLE
feat(examples): add e-commerce widgets demo styles [part 4]

### PR DESCRIPTION
**Pull request by francoischalifour**
_This is part 4 (final for the MVP) of the series "e-commerce demo"._

## Description

This adds the demo-specific widget styles to get the final look and feel.

## Notes

- The styles are alphabetically ordered

## Preview

[See preview →](http://deploy-preview-3802--instantsearchjs.netlify.com/examples/e-commerce/)

## Related

- [E-commerce demo design](https://app.zeplin.io/project/5acc8becfc0c980068fc12cc/screen/5cd1a7992e4f02039b5a8336)

### CODE REVIEW

**Comment by Haroenv** [comment]
Screenshot 2019-05-21 at 16 34 19

(placeholder) text isn't centred on the cursor it seems

**Comment by eunjae-lee** [comment]
Kapture 2019-05-21 at 17 01 07

The height of the magnifier icon is 16, but the height of the button is 20, which makes the magnifier vertically not centered.

**Comment by francoischalifour** [comment]
@tkrugg @Haroenv It was also coming from the "Hind" font that we use which seems to not be exactly vertically centered. This should now be fixed.

File: website/src/e-commerce/src/app.css [hunk 1]
```diff
@@ -128,9 +128,13 @@ mark {
   border-radius: 8px;
   box-shadow: 0 4px 48px 0 rgba(0, 0, 0, 0.2);
   font-family: Hind, sans-serif;
-  max-height: 64px;
-  padding: 24px;
+  height: 64px;
   padding-left: 58px;
+  /*
+    The "Hind" font family is vertically off-balance.
```
[the remark this answers is not in the document]
Comment (francoischalifour) [reply website/src/e-commerce/src/app.css]: Hind is only used in the demo styles, not in the theme itself. This is part of the design that we were given.

File: examples/e-commerce/src/app.css [hunk 2]
```diff
@@ -242,7 +242,12 @@ mark {
 }
 
 .range-input-submit {
-  display: none;
+  /*
```
Comment (eunjae-lee) [inline examples/e-commerce/src/app.css]: 👍

**Pull request merged**